src/components/DropdownMenu/index.tsx

```diff
@@ -1,14 +1,14 @@
 import {CSSProperties, MouseEvent, ReactNode, useCallback, useRef, useState} from 'react';
 import {createPortal} from 'react-dom';
-
+import * as S from './Styles';
```

Comment:
I may be wrong, but I have mostly seen this import: `import * as S from './Styles';` at the end of all imports